aptos-move/e2e-tests/src/executor.rs

```diff
@@ -760,7 +763,7 @@ impl FakeExecutor {
     pub fn new_block_with_timestamp(&mut self, time_microseconds: u64) {
         self.block_time = time_microseconds;
 
-        let validator_set = ValidatorSet::fetch_config(&self.data_store.as_move_resolver())
```

Comment:
so as_move_resolver still implemented stateview interfaces?

Reply:
Here we have `OnChainConfig` being implemented for `S: StateView`, so call `as_move_resolver` is not needed